### ISSUE TITLE
'stop' doesn't immediately stop playback

### ISSUE THREAD

**Issue by tcs-devel** [goldfire/howler.js#1490]
In rare circumstances we are experiencing an issue where a call to `stop()` function does not immediately stop playback, the action then seems to be executed only after the audio playback has ended.

Unfortunately we do not yet have a code snipped to repro the issue, just a view hints. We are using HTML5 (not pre-loaded).
From debugging we saw the `_playLock` is set because the code reaches `// Implements a lock to prevent DOMException: The play() request was interrupted by a call to pause().`
If then `stop()` function is invoked, the stop action is enqueued because `_playlock` ist set to `true`; in this case the playback is then not stopped immediately, but only after the media has been fully played.
 
To summarize  the steps:
1) invoke `play()`
  causing `_playLock` to be set to true
 https://github.com/goldfire/howler.js/blob/4957be0dd1b473ef94e3db260914bebed2a577d6/src/howler.core.js#L911
2) invoke `stop()` while the promise is not resolved
 causing "stop" is enqueued because `_playLock` is true
 https://github.com/goldfire/howler.js/blob/4957be0dd1b473ef94e3db260914bebed2a577d6/src/howler.core.js#L1082

**Comment by EvidentlyCube** [comment]
I'm experiencing the same problem when making a webpack production build of my project but not in development. Still trying to figure out what is happening.
